hazelcast/src/main/java/com/hazelcast/core/MapLoader.java

```diff
@@ -49,6 +49,9 @@
      * Loads given keys. This is batch load operation so that implementation can
      * optimize the multiple loads.
      *
+     * For any key in the input keys, there should be a single mapping in the resulting map. Also the resulting
+     * map should not have any keys that are not part of the input keys.
+     *
      * @param keys keys of the values entries to load
      * @return map of loaded key-value pairs.
```

Comment:
Could you also add the passed keys should not be null?
It applies to both methods load, loadAll.

Also loadAllKeys should not return nulls.